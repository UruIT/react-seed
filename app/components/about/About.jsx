import React from 'react';

const About = () => (
	<div>
		<h1>About</h1>
	</div>
);

export default About;
