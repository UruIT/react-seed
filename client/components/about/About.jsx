import React from 'react';
import Counter from '../counter';
import styles from './about.scss';

const About = () => (
	<div className={styles.about}>
		<h1>React Seed</h1>
		<h2>Simple project to start with React!</h2>
		<Counter />
	</div>
);

export default About;
