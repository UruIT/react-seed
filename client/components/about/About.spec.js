import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import About from './About';

describe('<About/>', () => {
	it('snapshot', () => {
		const renderer = new ShallowRenderer();
		const tree = renderer.render(<About />);
		expect(tree).toMatchSnapshot();
	});
});
