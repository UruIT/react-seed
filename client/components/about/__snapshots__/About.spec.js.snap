// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About 1`] = `
<div
  className="about"
>
  <h1>
    About
  </h1>
</div>
`;
